src/main/java/pl/piekoszek/gorskimatches/token/EmailService.java

```diff
@@ -0,0 +1,33 @@
+package pl.piekoszek.gorskimatches.token;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Component;
+
+import javax.mail.MessagingException;
+import java.util.Properties;
+
+
+@Component
```

Comment:
This class looks like configuration, not service. Use `@Configuration` annotation instead. 

Also mail should work without it. Config in `application.properties` is enough.

Reply:
ok